atd-vzv/src/views/summary/FatalitiesByMode.js

```diff
@@ -56,8 +70,14 @@ const FatalitiesByMode = () => {
 
   // Fetch latest record from demographics dataset and set for chart subheading
   useEffect(() => {
-    const url = `${demographicsEndpointUrl}?$limit=1&$order=crash_date DESC&$where=crash_date < '${thisYear}-12-31T23:59:59'`;
-
+    let url;
+    // If it is past January, set end of query to last day of previous month,
+    // else if it is January, set end of query to last day of last year
+    if (thisMonth > "01") {
+      url = `${demographicsEndpointUrl}?$limit=1&$order=crash_date DESC&$where=crash_date < '${thisYear}-${lastMonth}-${lastDayOfLastMonth}T23:59:59'`;
+    } else {
+      url = `${demographicsEndpointUrl}?$limit=1&$order=crash_date DESC&$where=crash_date < '${lastYear}-12-31T23:59:59'`;
+    }
```

Comment:
I think instead of doing the repetitive var setting for url inside the if/else here, you could just if else the endDate itself. Ex:

```js
let endDate = thisMonth > "01" 
   ? `${thisYear}-${lastMonth}-${lastDayOfLastMonth}T23:59:59` 
   : `${lastYear}-12-31T23:59:59`;

let url = `${demographicsEndpointUrl}?$limit=1&$order=crash_date DESC&$where=crash_date < ${endDate}`;
```

Maybe thats more a personal preferences that actually more readable.

Reply:
I like that! Definitely more eloquent. I'll make the change tomorrow morning and merge.

Reply:
Replaced this instance and two others with the ternary operation you suggested